fares-data-build-tool/create-data
# feat: Editing products should default

## Description

All edit functions in the product details page should offer default values previously defined for product 

## Testing instructions

Check services, other products and multi op pages try and edit some product attributes for different ticket types
